crates/solver/src/settlement_submission/submitter/eden_api.rs

```diff
@@ -37,9 +69,16 @@ impl TransactionSubmitting for EdenApi {
         &self,
         tx: TransactionBuilder<Web3Transport>,
     ) -> Result<TransactionHandle, SubmitApiError> {
-        self.rpc
-            .api::<PrivateNetwork>()
-            .submit_raw_transaction(tx)
+        // try to submit with slot method
+        self.submit_slot_transaction(tx.clone())
```

Comment:
Does this fail if you don't have a valid slot?

Reply:
I _think_ it shouldn't fail because when I send a dummy signed tx it returns the hash. But I wasn't really confident in this new function (these RPCs often tend to disrespect the json standard and similar) so wanted to keep the old gold `eth_sendRawTransaction` in case of any kind of error with `submit_slot_transaction`.
How about I add a comment about this in the code so we can remove the fallback call after some time if not needed, wdyt?

Reply:
Comment would be great!

Reply:
Resolved with comment.